docs/src/main/mdoc/modules.md

```diff
@@ -89,6 +89,17 @@ object Color extends StringEnum[Color] with StringCirisEnum[Color] {
 ConfigDecoder[String, Color]
 ```
 
+## Http4s
+
+The `@HTTP4S_MODULE_NAME@` module provides [`ConfigDecoder`][configdecoder]s for [http4s](https://github.com/http4s/http4s) Uri type.
```

Comment:
I have specified `http4s Uri type`, not `http4s types`.
I don't see any other suitable type at the moment, so specifying `Uri type` here.
Maybe in the future we will add more decoders for `http4s` related types, and this will be changed, but for now (as I don't know any other suitable types), I would like to be precise.

Reply:
Agreed. I changed the wording very slightly, but kept `Uri` specifically.